k8s-deploy/pkg/api/chart.go

```diff
@@ -47,6 +48,35 @@ type HelmUUID struct {
 	HelmUUID string
 }
 
+// getValueTemplateExample return valueTemplateExample string by chart name and version
+// @Summary List all historical Charts
+// @Tags Chart
+// @Produce  json
+// @Success 200 {object} JSONResult{data=modules.HelmChart} "Success"
+// @Failure 401 {object} JSONERRORResult "Unauthorized operation"
+// @Failure 500 {object} JSONERRORResult "Internal server error"
+// @Router /chart [patch]
```

Comment:
Perhaps this is the reason?

Reply:
I mean the reason why it is still patch.

I think you need to check these comments carefully.

Reply:
ok，I will fix